src/test/kotlin/lottery/domain/InputValidatorTest.kt

```diff
@@ -0,0 +1,43 @@
+package lottery.domain
+
+import io.kotest.matchers.shouldBe
+import lottery.validator.InputValidator
+import org.assertj.core.api.Assertions
+import org.junit.jupiter.api.DisplayName
+import org.junit.jupiter.api.Test
+
+class InputValidatorTest {
+
+    @Test
+    @DisplayName("로또 구입 금액이 숫자가 아니면 구매가 불가능하다")
+    fun validateAmount() {
```

Comment:
JUnit5에서는 다음과 같이 작성할 수 있습니다!
```suggestion
    fun `로또 구입 금액이 숫자가 아니면 구매가 불가능하다`() {
```

Reply:
빠르게 모든 메서드를 한글로 변경했습니다! 더 보기 좋네요 😄